lib.rs

```diff
@@ -315,6 +315,29 @@ impl<A: Array> SmallVec<A> {
     }
 }
 
+impl<A: Array> SmallVec<A> where A::Item: Copy {
+    pub fn insert_slice(&mut self, index: usize, slice: &[A::Item]) {
+        self.reserve(slice.len());
```

Comment:
Oh, you're right, nvm, I thought `reserve()` did the usual stuff, that is, ensuring capacity.